samples/LottieSamples/LottieSamples.csproj

```
@@ -261,7 +261,7 @@
       <Version>6.2.9</Version>
     </PackageReference>
     <PackageReference Include="Microsoft.Toolkit.Uwp.UI.Lottie">
-      <Version>5.1.1</Version>
+      <Version>6.0.0</Version>
```

```suggestion
      <Version>[6.0.0-*,6.0.0)</Version>
```
Can't reference the unshipped package yet, but this may pick-up the rc one instead in the meantime.